.github/workflows/release-defaults.yml

```
@@ -0,0 +1,33 @@
+name: Upload files to the GitHub release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    name: Publish ${{ matrix.file_name }}
+    runs-on: ${{ matrix.file_name }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            file_name: .scalafix.conf
+          - os: ubuntu-latest
+            file_name: .scalafmt.conf
+
+    steps:
+    - uses: hecrj/setup-rust-action@v1-release
+      with:
+        rust-version: stable
+    - uses: actions/checkout@v1
```

Use `actions/checkout@v2` if possible

```suggestion
    - uses: actions/checkout@v2
```